index.html

```diff
@@ -0,0 +1,110 @@
+            </div>
+            <div class="nav-link">
+                <a href="#Codecademy_is_the_fourth_one">Codecademy is the fourth one</a>
+            </div>
+
+        </nav><br><br>
+        <div id="technical-doc-page">
+            <section class="main-section">
+                <header>4 best (free) online coding courses for beginners</header>
+                <p>The prospect of learning how to code can be overwhelming, especially if you don’t have any computer
+                    science knowledge or experience.</p>
+
+                <p>But software engineering, data science, and related fields can be stable, long-term career paths no
+                    matter where you live — there’s a huge demand right now for employees with computer science skills
+                    at established companies and startups.</p>
+                <p>Plus, jobs in these fields tend to pay well and come with lots of associated perks and benefits.</p>
+                <p>If you’re mulling over the idea of a new career, learning how to code is a great investment in
+                    yourself.</p>
+            </section>
+
+            <section class="main-section">
+                <header>Galvanize is the first one</header>
+                <p>If you’re interested in learning one of the most in-demand coding languages on the market — Python —
+                    then look no further than Galvanize’s data science free prep program.</p>
+                <ul>
+                    <li>w3schools</li>
+                    <li>hackerrank</li>
+                    <li>microverse</li>
+                    <li>codehs</li>
+                    <li>Microverse</li>
+                </ul>
+                <p>If you have dreams of someday becoming a data scientist, this course will help you get up to speed on
+                    statistics and the fundamentals of Python.</p>
+            </section>
+
+            <section class="main-section">
+                <header>Hack Reactor is the second</header>
+                <p>Hack Reactor’s online coding bootcamp is the best around, so it’s no surprise that the company’s
+                    software engineer free prep program is also incredibly effective for learning Javascript as a
+                    beginner.</p>
+                <p>Hack Reactor offers a free, online, self-guided prep course that lets you study at your own pace and
+                    work on your own time.</p>
+            </section>
+
+            <section class="main-section">
+                <header>Udemy is the third</header>
+                <p>Udemy is a cool online platform that offers more than 100,000 courses on a whole host of subjects,
+                    including coding. The platform has a huge variety of free programming fundamentals courses, so you
+                    can pick and choose or stack courses back-to-back based on your interests.</p>
+            </section>
+
+            <section class="main-section">
```

Comment:
Class used without being created in the CSS file. Attention!

Reply:
same here, an explanation on why this is bad is also needed